### ISSUE TITLE
[Feature Request]Ability to hide certain apps in launcher such as Charge Animation.

### ISSUE THREAD

**Issue by Ronin0000** [espruino/BangleApps#1560]
It would be very nice to be able to hide certain apps such as the Charge Animation. I might add this feature if I have time.

**Comment by Ronin0000** [comment]
I will work on this feature!

**Comment by gfwilliams** [comment]
Thanks!

**Comment by Chriz76** [comment]
Are there already ideas how to implement this? Just a (documented) metadata setting and then filtering it in the default launcher?

**Comment by thyttan** [comment]
What apps do you want to hide? If they should be labeled as 'bootloader' apps that would solve it.

**Comment by Chriz76** [comment]
> What apps do you want to hide? If they should be labeled as 'bootloader' apps that would solve it.

Yes, looks like it is that easy. Just use a category other than 'app'. @Ronin0000 what do you think?

**Comment by gfwilliams** [comment]
> Just use a category other than 'app'.

Yes, that's the easiest - ideally we just do a PR to change the relevant apps for everyone.

If you needed it to be configurable on a per-user basis I guess we could either have something that ran and reconfigured the app.info files (but they would change back each time there was a new version), or I guess all launchers could be rewritten to use a `getLauncherApps` function in a module in `modules` and then maybe something more fancy could be added there - but if we can get away with changing `info` files that'd be simpler IMO

**Comment by gfwilliams** [comment]
Closing as inactive - but as mentioned it's easy to not have apps appear in the launcher. Just set type to something other than `type:"app"` in the metadata - available types are all documented in https://github.com/espruino/BangleApps/blob/master/README.md#metadatajson-format
